### PR TITLE
create index.html

### PR DESCRIPTION
**Pull request by apolyvka**
https://github.com/apolyvka/studious-guacamole/pull/1#issue-2017160209

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```
Comment (loralevitska) [inline index.html:1]: Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)

**Review by loralevitska — CHANGES_REQUESTED** [review]
Provide `My page` link to PR description according task requirement 
image
image

**Comment by apolyvka** [comment]
https://apolyvka.github.io/studious-guacamole/

**Review by andriy-shymkiv — APPROVED** [review]
good job!

**Pull request merged**